starters/http/dotnet/infra/core/host/functions-flexconsumption.bicep

```diff
@@ -11,6 +11,7 @@ param virtualNetworkSubnetId string = ''
 param identityType string
 @description('User assigned identity name')
 param identityId string
+param identityClientId string
```

Comment:
My instincts are to avoid passing yet another identity parameter in favor of converting or rehydrating the identity object from the id.  Looking for inspiration in bicep universe first..

Reply:
ok a different idea is to set the app insights client id in processor.bicep, which is also what we do for other client ids.  this keeps UAMI specific/required concepts out of the core module and achieves the same outcome.  What do you think?  See SB and EH e2e samples that have client ids.

Reply:
Fixed as discussed